rdt-open-reader/src/covidIndonesia/java/io/ona/rdt/fragment/CovidPatientRegisterFragment.java

```diff
@@ -42,4 +46,16 @@ protected String getPatientRegistrationForm() {
     protected String getDefaultSortQuery() {
         return Constants.DBConstants.LAST_INTERACTED_WITH + " DESC";
     }
+
+    @Override
+    public void onSyncComplete(FetchStatus fetchStatus) {
+        super.onSyncComplete(fetchStatus);
+        RDTApplication.getInstance()
+                .getContext()
+                .allSharedPreferences()
+                .getPreferences()
+                .edit()
+                .putLong(CovidConstants.SyncPreference.LAST_SYNC_DATE, System.currentTimeMillis())
+                .apply();
```

Comment:
We cannot use this key `CovidConstants.SyncPreference.LAST_SYNC_DATE` since it overrides the sync value from the server. Rename this to `LATEST_SYNC_TIMESTAMP`.

Also, you can use `.savePreference()` for simplicity.

Reply:
done